### PR TITLE
Update aerospike chart

### PR DESCRIPTION
**Pull request by moeryomenko**
#### Which issue this PR fixes
  - TolerateUnreadyEndpoints deprecation in 1.10
  - Upgrade Aerospike version

#### Checklist
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by moeryomenko** [comment]
@cpanato @paulczar pls approve

File: stable/aerospike/Chart.yaml [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
 home: http://aerospike.com
-version: 0.2.1
+version: 0.2.3
 icon: https://s3-us-west-1.amazonaws.com/aerospike-fd/wp-content/uploads/2016/06/Aerospike_square_logo.png
 sources:
 - https://github.com/aerospike/aerospike-server
 maintainers:
 - name: kavehmz
   email: kavehmz@gmail.com
+- name: moeryomenko
```
Comment (cpanato) [inline stable/aerospike/Chart.yaml]: to add a new maintainer please first talk with the existing ones and one another pr for that

Comment (moeryomenko) [reply stable/aerospike/Chart.yaml]: remove

**Comment by moeryomenko** [comment]
@cpanato pls check

**Pull request merged**